.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,70 @@
+name: Run tests
+
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - '*'
+  pull_request:
+
+
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Code style checks
+            os: ubuntu-latest
+            python: 3.x
+            toxenv: codestyle
+
+          - name: Build the docs
+            os: ubuntu-latest
+            python: 3.8
+            toxenv: build_docs
+
+          - name: Python 3.9 with minimal dependencies
+            os: ubuntu-latest
+            python: 3.9
```

Comment:
Should this always be the latest python version, so we don't need to update it annually? Can we do that with `python: 3.x`?

Reply:
I think that we want to do this manually. GitHub might bump their runner up to the next version before our dependencies have fully migrated over leading to test failures that aren't the fault of sncosmo.